File: java/aula04/Jogador.java
public class Jogador {
    
    private String nome;
    private String posicao;
    private double peso;
    private double altura;
    private double velocidade;

    Jogador(String nome, String posicao, double peso, double altura, double velocidade){
        this.nome = nome;
        this.posicao = posicao;
        this.peso = peso;
        this.altura = altura;
        this.velocidade = velocidade;
    }

    public String toString(){
        return "Jogador: " + this.nome + " jogando na posição:" + this.posicao;
    }

    public String getNome(){
        return this.nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getPosicao() {
        return posicao;
    }

    public void setPosicao(String posicao) {
        this.posicao = posicao;
    }

    public double getPeso() {
        return peso;
    }

    public void setPeso(double peso) {
        this.peso = peso;
    }

    public double getAltura() {
        return altura;
    }

    public void setAltura(double altura) {
        this.altura = altura;
    }

    public double getVelocidade() {
        return velocidade;
    }

    public void setVelocidade(double velocidade) {
        this.velocidade = velocidade;
    }
}

File: java/aula04/jogador.py
class Jogador:
    """ Esta é a classe jogador"""
    __nome = "" 
    __posicao = ""
    __peso = 0.0
    __altura = 0.0
    __velocidade = 0.0

    def __init__(self,nome,posicao,peso,altura,velocidade):
        self.__nome = nome
        self.__posicao = posicao
        self.__peso = peso
        self.__altura = altura
        self.__velocidade = velocidade


    def getVelocidade(self):
        return self.__velocidade
    
    def setVelocidade(self,velocidade):
        self.__velocidade = velocidade

    def getPeso(self):
        return self.__peso

    def getAltura(self):
        return self.__altura
    
    def getPosicao(self):
        return self.__posicao 
    
    def getNome(self):
        return self.__nome